website/docs/r/app_service.html.markdown

```diff
@@ -170,6 +176,15 @@ The following attributes are exported:
 
 * `outbound_ip_addresses` - A comma separated list of outbound IP addresses - such as `52.23.25.3,52.143.43.12`
 
+* `source_control` - (Optional) The default local Git source control information if deployment option is set to `LocalGit`.
+
+---
+
+`source_control` supports the following:
+
+* `repo_url` - URL of the Git repo.
+* `branch` - Branch name of the Git repo.
```

Comment:
can we suffix both of these with `.. the Git repository for this App Service`

Reply:
Done.